### DISCUSSION TITLE
Inconsistency on `allowedProtocols`

### DISCUSSION THREAD

**Discussion by parallels999** [dompdf/dompdf#3326] [Q&A] [answered]
Shouldn't it also support `allowed_protocols` key for consistency?
```php
} elseif ($key === 'allowedProtocols' || $key === 'allowed_protocols') { 
   $this->setAllowedProtocols($value); 
}
```
https://github.com/dompdf/dompdf/blob/9dce24113c6fd7d0a42c737fc0c68652777a9e77/src/Options.php#L362-L370

**Comment by bsweeney** [comment, marked as answer]
Sure, I think that makes sense.
